src/text_buffer.rs

```diff
@@ -130,7 +135,8 @@ impl TextBuffer {
             cy: 0,
             file,
             row,
-            modified: false,
+            modified_count: 0,
+            tmp_modified: false,
```

Comment:
What is the reason to remain `modified` as `tmp_modified`? I thought replacing `modified` with `modified_count` is simpler

Reply:
`tmp_modified` is not same meaning as `modified`.
`tmp_modified` indicates that there are some changes not put into history.
In the case, `modified_count` might be `0`, but
I want to display `(modified)` in status bar.

Reply:
I see. So I guess 'temporary' is similar to 'ongoing' here.